rllib/offline/offline_evaluation_runner.py

```diff
@@ -456,7 +458,7 @@ def _batch_iterator(self) -> MiniBatchRayDataIterator:
         return self.__batch_iterator
 
     @property
-    def _device(self) -> DeviceType:
+    def _device(self) -> Union[DeviceType, None]:
```

Comment:
hey @simonsays1980 is `None` allowed here?

Reply:
Yes, this can practically not happen, but theoretically. We left some doors open here if we would ever turn to another framework like JAX, we do not need to have to much hassle to integrate it. Great Fix!